src/rpc/rawtransaction.cpp

```diff
@@ -1740,40 +1798,46 @@ static RPCHelpMan joinpsbts()
         throw JSONRPCError(RPC_INVALID_PARAMETER, "At least two PSBTs are required to join PSBTs.");
     }
 
-    uint32_t best_version = 1;
+    int32_t best_version = 1;
```

Comment:
Why change to a signed int? IIUC this is checked against the version field in transaction which is unsigned.

Reply:
It was unsigned when this PR was written 3 years ago lol.

Updated to be unsigned.

Reply:
Nice!